src/plugins/aragon/index.ts

```diff
@@ -193,50 +193,46 @@ async function scheduleAction(
 ) {
   const query = GQL_QUERY;
   query.registryEntry.__args.id = daoName;
-  const { registryEntry } = await subgraphRequest(
-    ARAGON_SUBGRAPH_URL[4],
-    query
-  );
+  const result = await subgraphRequest(ARAGON_SUBGRAPH_URL['4'], query);
+  const config = result.registryEntry.queue.config;
 
   // Building the nonce for the next tx
-  const nonce = await web3.nonce();
+  const nonce = await web3.getTransactionCount(account, 'pending');
```

Comment:
I'm not exactly sure this might work—we must get the nonce from the queueContract itself; it's a variable. See [the console code](https://github.com/aragon/govern/blob/master/packages/govern-console/src/components/NewAction.tsx#L237). This means that we will need the `GovernQueue` address from the query to instanciate the contract and view the `nonce` variable.